src/utils/copyExternalArtifacts.ts

```diff
@@ -12,10 +12,7 @@ export function copyExternalArtifacts(
 ): void {
   const fromDir = path.resolve(externalArtifactsPath)
   const toDir = path.resolve(localArtifactsPath)
-  if (!fs.existsSync(fromDir))
-    throw Error(
-      `Can not copy external artifacts from ${fromDir}, does not exist`
-    )
+  if (!fs.existsSync(fromDir)) return //fail silently
```

Comment:
Maybe we can detect if the `fromDir` is one of `externalArtifactPaths` and only not throw then?

I imagine this is a user-configured option and we should alert if they set it to something else that we can't find.